test/test_helper.exs

```diff
@@ -21,7 +21,12 @@ defmodule XandraTest.IntegrationCase do
             default_start_options: @default_start_options
           ] do
       setup_all do
-        keyspace = "xandra_test_" <> String.replace(inspect(__MODULE__), ".", "")
+        module_suffix =
+          inspect(__MODULE__)
+          |> String.replace(".", "")
+          |> String.downcase()
+
+        keyspace = "xandra_test_" <> module_suffix
```

Comment:
Why is this necessary? Can't we compare with `String.downcase/1` from the query result in the new tests?

Reply:
@whatyouhide as I explained in the commit message, I changed the test helper because the keyspace actually gets created lowercased (see https://docs.datastax.com/en/cql-oss/3.x/cql/cql_reference/ucase-lcase_r.html). The other tests didn't care because the name just gets passed as is and is translated by the DB automatically, but matching explicitly on the returned keyspace name revealed the inconsistency.

Of course I can just change this just in my test, I just thought that it was better to fix the inconsistency at the source so that tests can use the correct created keyspace name, let me know what I should do.